tools/goctl/api/gogen/gen_test.go

```diff
@@ -53,7 +54,10 @@ func TestParser(t *testing.T) {
 	filename := "greet.api"
 	err := os.WriteFile(filename, []byte(testApiTemplate), os.ModePerm)
 	assert.Nil(t, err)
-	defer os.Remove(filename)
+	env.Set(t, env.GoctlExperimental, "off")
```

Comment:
t.Setenv("ENV_VAR", "value")

Reply:
Here is not system env, it's goctl env which wrapped by file system.